for-research/public-likelihoods/public-likelihoods.ipynb

```diff
@@ -0,0 +1,936 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ddebfcc8-55fe-47bd-9a4b-3b5afa67c167",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Warning in <xRooNode::xRooNode>: Inferring initial errors of 116 parameters (give all nuisance parameters an error to avoid this msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "w = ROOT.xRooNode(\"Unfolding_dPhi_ttbar_Z_5bins_comb_particle_tau2.4_NORM.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64a98d0e-a16f-40c4-8e92-5b5bbae081ed",
+   "metadata": {},
+   "source": [
+    "You can explore the content of the workspace somewhat like you would a file system: each node contains sub-nodes, which you can interact with to explore ever deeper. Here are some methods we"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3fb95240-831d-4af3-a2d1-aae9fee9b0a6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "This workspace contains 2 pdfs ( ['regularization_dPhi_ttbar_Z', 'simPdf'] ) and 2 datasets ( ['asimovData', 'obsData'] )\n",
+      "There are 470 parameters and 360 observables (of which 350 are global and 10 are regular)\n",
+      "There are currently 357 floating parameters\n",
+      "There are 4 parameters of interest:\n",
+      "Unfolding_dPhi_ttbar_Z_5bins_comb_particle_tau2.4_NORM.json/.poi: RooArgList::Unfolding_dPhi_ttbar_Z_5bins_comb_particle_tau2.4_NORM.json.poi\n",
+      "0) dPhi_ttbar_Z_Bin_001_mu : RooRealVar::dPhi_ttbar_Z_Bin_001_mu = 1  L(0 - 2) \n",
+      "1) dPhi_ttbar_Z_Bin_002_mu : RooRealVar::dPhi_ttbar_Z_Bin_002_mu = 1  L(0 - 2) \n",
+      "2) dPhi_ttbar_Z_Bin_003_mu : RooRealVar::dPhi_ttbar_Z_Bin_003_mu = 1  L(0 - 2) \n",
+      "3) dPhi_ttbar_Z_Bin_004_mu : RooRealVar::dPhi_ttbar_Z_Bin_004_mu = 1  L(0 - 2) \n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"This workspace contains\",w[\"models\"].size(),\"pdfs (\", \n",
+    "      [m.GetName() for m in w[\"models\"]], \") and\",w[\"datasets\"].size(),\"datasets (\",\n",
+    "      [d.GetName() for d in w[\"datasets\"]],\")\")\n",
+    "print(\"There are\", w.pars().size(), \"parameters and\",w.obs().size(),\"observables (of which\", \n",
+    "      w.globs().size(),\"are global and\",w.robs().size(),\"are regular)\")\n",
+    "print(f\"There are currently {w.floats().size()} floating parameters\")\n",
+    "print(f\"There are {w.poi().size()} parameters of interest:\")\n",
+    "w.poi().Print()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b019e58-89a8-416c-8121-9ce94c61fc3f",
+   "metadata": {},
+   "source": [
+    "Note that subnodes can be accessed either by name or index. \n",
+    "\n",
+    "## 1.2. Accessing bin contents and uncertainties\n",
+    "\n",
+    "Let us now explore the contents and uncertainties of a pdf in the workspace:"
```

Comment:
Again, if you've got time, I'd love to see more explanation of what comes next. "The fit is divided into different histograms, called channels, which might each have a different number of bins. With each channel..."